todo/services/task_service.py

```
@@ -148,6 +153,19 @@ def _prepare_label_dtos(cls, label_ids: List[str]) -> List[LabelDTO]:
             for label_model in label_models
         ]
 
+    @classmethod
+    def prepare_deferred_details_dto(cls, deferred_details_model: DeferredDetailsModel) -> DeferredDetailsDTO | None:
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)